types/convert.go

```diff
@@ -275,7 +275,8 @@ func floatStrToIntStr(validFloat string) (string, error) {
 	}
 	if exp > 0 && int64(intCnt) > (math.MaxInt64-int64(exp)) {
 		// (exp + incCnt) overflows MaxInt64.
-		return validFloat, ErrOverflow.GenByArgs("BIGINT", validFloat)
+		sc.AppendError(ErrOverflow.GenByArgs("BIGINT", oriStr))
```

Comment:
This means the previous error message is wrong?

Reply:
yes, the error message in this case is like below

mysql> show warnings;
+---------+------+-------------------------------------------------+
| Level   | Code | Message                                         |
+---------+------+-------------------------------------------------+
| Warning | 1292 | Truncated incorrect INTEGER value: '125e342.83' |
+---------+------+-------------------------------------------------+

Reply:
So why `AppendError`? Shouldn't we `AppendWarning`?

Reply:
yes, it's should be AppendWarning. Already Fixed